### PR TITLE
Automatic port of System.Runtime docs

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: xml/System.Runtime.CompilerServices/ModuleInitializerAttribute.xml [hunk 1]
```diff
@@ -20,8 +20,25 @@
...
+When one or more valid methods with this attribute are found in a compilation, the compiler will emit a module initializer which calls each of the attributed methods.
+             Certain requirements are imposed on any method targeted with this attribute:
+             - The method must be `static`.
+             - The method must be an ordinary member method, as opposed to a property accessor, constructor, local function, etc.
+             - The method must be parameterless.
+             - The method must return `void`.
+             - The method must not be generic or be contained in a generic type.
+             - The method's effective accessibility must be `internal` or `public`.
+             The specification for module initializers in the .NET runtime can be found here:
+             https://github.com/dotnet/runtime/blob/master/docs/design/specs/Ecma-335-Augments.md#module-initializer
```
Comment (carlossanlop) [inline xml/System.Runtime.CompilerServices/ModuleInitializerAttribute.xml:38]:
```suggestion
             The specification for module initializers in the .NET runtime can be found [here](
             https://github.com/dotnet/runtime/blob/master/docs/design/specs/Ecma-335-Augments.md#module-initializer).
```

**Review by pgovind — APPROVED** [review]

File: xml/System.Runtime.CompilerServices/RuntimeFeature.xml [hunk 2]
```diff
@@ -223,7 +223,7 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that this version of runtime supports the Unmanaged calling convention value.</summary>
```
Comment (gewarren) [inline xml/System.Runtime.CompilerServices/RuntimeFeature.xml:226]:
```suggestion
        <summary>Represents a runtime feature where C# function pointers can be declared with an unmanaged calling convention.</summary>
```

File: xml/System.Runtime.CompilerServices/ModuleInitializerAttribute.xml [hunk 3]
```diff
@@ -20,8 +20,24 @@
...
+
+When one or more valid methods with this attribute are found in a compilation, the compiler will emit a module initializer which calls each of the attributed methods.
+             Certain requirements are imposed on any method targeted with this attribute:
+             - The method must be `static`.
+             - The method must be an ordinary member method, as opposed to a property accessor, constructor, local function, etc.
+             - The method must be parameterless.
+             - The method must return `void`.
+             - The method must not be generic or be contained in a generic type.
+             - The method's effective accessibility must be `internal` or `public`.
+             The specification for module initializers in the .NET runtime can be found [here](https://github.com/dotnet/runtime/blob/master/docs/design/specs/Ecma-335-Augments.md#module-initializer).
```
Comment (gewarren) [inline xml/System.Runtime.CompilerServices/ModuleInitializerAttribute.xml:37]:
```suggestion

For more information, see [the specification for module initializers in the .NET runtime](https://github.com/dotnet/runtime/blob/master/docs/design/specs/Ecma-335-Augments.md#module-initializer).
```

**Comment by gewarren** [comment]
@pgovind Since this is already merged, can you take a look at my suggested changes? If they look good to you, I'll add them in a new PR.

**Comment by pgovind** [comment]
@gewarren : Yup, those edits sound much much better!

**Pull request merged**